package exception.runtimeexception.test;

public class RuntimeExceptionTest {

	public static void main(String[] args) {
		
		int a = 10;
		
	}
}
